spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -1,5 +1,62 @@
 require 'rails_helper'
 
+shared_examples 'the idv/doc_auth session is cleared' do
+  it 'clears the session' do
+    expect(controller.user_session['idv/doc_auth']).to be_blank
+  end
+end
+
+shared_examples 'the idv/in_person session is cleared' do
+  it 'clears the session' do
+    expect(controller.user_session['idv/in_person']).to be_blank
+  end
+end
+
+shared_examples 'the idv/inherited_proofing session is cleared' do
+  it 'clears the session' do
+    expect(controller.user_session['idv/inherited_proofing']).to be_blank
+  end
+end
+
+shared_examples 'the decrypted_pii session is cleared' do
+  it 'clears the session' do
+    expect(controller.user_session[:decrypted_pii]).to be_blank
+  end
+end
+
+shared_examples 'a redirect occurs to the start of identity verification' do
+  it 'redirects' do
+    delete :destroy
+
+    expect(response).to redirect_to(idv_url)
+  end
+end
+
+shared_examples 'logs IDV start over analytics with step and location params' do
+  it 'logs the analytics' do
+    delete :destroy, params: { step: 'first', location: 'get_help' }
+
+    expect(@analytics).to have_logged_event(
+      'IdV: start over',
+      step: 'first',
+      location: 'get_help',
+    )
+  end
+end
+
+# LG-7128: Implement Inherited Proofing analytics here.
+shared_examples 'logs Inherited Proofing start over analytics with step and location params' do
+  xit 'logs the analytics' do
+    delete :destroy, params: { step: 'first', location: 'get_help' }
+
+    expect(@analytics).to have_logged_event(
+      'InheritedProofing: start over',
+      step: 'first',
+      location: 'get_help',
+    )
+  end
```

Comment:
As a follow-on to our offline chat about skipped tests, personally I don't expect to see any `xit` and would prefer they not exist at all, until we're implementing the behavior we want to test. Even if it could prove useful for a starting point for future iteration, I think it's quite likely to fall out of sync, may not align to how we choose to implement in the future, and it's hard to hold ourselves accountable to eventually doing something with it (our other existing `xit` are good examples of this).

Reply:
Done.